docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,740 @@
+
+.. image:: ../_static/control_center_pics/multi_module_serial.png
+
+We highly recommend connecting your modules together with a non-permanent, easily detachable method. Some configurations through the Control Center 
+must be performed on independently connected modules before multiple can be connected at once. So, having a quick way to attach/detach modules will 
+make the process much easier. A simple way to accomplish this is with 3-pin, 100-mil header pins and a breadboard.
+
+In order to connect with several modules at once, you will also have to configure each module's Module ID. 
+More information about this can be found :ref:`below <multi_module_config>`.
+
+.. _gui_overview:
+
+***********************************
+IQ Control Center GUI Overview
+***********************************
+.. image:: ../_static/control_center_pics/gui_breakdown.png
+    :scale: 50%
+
+There are 5 major sections of the Control Center interface, which have been highlighted
+with colored boxes in the figure above and are described below:
+
+* The :red:`Tabs` section lists the different tabs of the Control Center that are available when connected to a module. Each tab offers access to different configuration parameters and functionality. The settings in these tabs will be covered in greater detail later.
+
+* The :green:`Information` section lists information about the Control Center and connected module, if there is one. When there is no module connected, this only lists the Control Center version. When a module is connected, this will also list its Firmware Version, Firmware Style, Hardware Style, Bootloader Version, and Upgrader Version. Check this information to make sure you are using the proper hardware with the latest firmware. You can check for new firmware under the Modules section of the Vertiq website.
+
+* The :blue:`Module Connections` section is used to connect to modules. You can select the baud rate and the serial port to use for the module you want to connect to. The serial port is determined by the name of the serial port used by your USB-to-UART converter. See the :ref:`serial_port` section for more details on how to determine that port name.
+
+* The :gold:`Display` section displays information, configuration parameters, and controls relevant to your current tab. When you change tabs, the controls and information shown here will change. This is where the majority of your interactions with a connected module will take place.
+
+* The :purple:`Messages` section displays information about the Control Center that can be useful for understanding what state the Control Center is in and debugging connection problems. Check here if you are having connection issues for feedback.
+
+**********************************
+Connecting With your Module(s)
+**********************************
+
+.. _multi_module_config:
+
+Required Configuration Before Connecting with Multiple Modules
```

Comment:
One thing that occurred to me, may be worth putting a note or warning here warning that some older firmwares may not support this. Like, the old 2306 servo firmware, pretty sure that is too old to support this feature